File: python/PlusMinus.py
def plusMinus(arr):
    # Write your code here
    lenth = len(arr)
    
    positiveCount = 0;
    negativeCount = 0;
    zeroCount = 0;
    
    for i in range(lenth):
        if(arr[i] > 0):
            positiveCount += 1;
        elif(arr[i] < 0):
            negativeCount += 1;
        elif(arr[i] == 0):
            zeroCount += 1;
    
    # print("{0:.6f}".format((positiveCount / lenth)));
    print("%0.6f "%(positiveCount / lenth));
    print("%0.6f "%(negativeCount / lenth));
    print("%0.6f "%(zeroCount / lenth));

File: javascript/PlusMinus.js
// Given an array of integers, calculate the ratios of its elements that are
// positive, negative, and zero. Print the decimal value of each fraction on a
// new line with  places after the decimal.


function plusMinus(arr) {
    // Write your code here
    
    let positive = 0;
    let negative = 0
    let zero = 0;
    
    for(let i = 0; i < arr.length; i++){
        if(arr[i] === 0){
            zero++;
        }else if(arr[i] >0 ){
            positive++;
        }else{
            negative++;
        }
    }
    
    console.log((positive/arr.length).toFixed(6))
    console.log((negative/arr.length).toFixed(6))
    console.log((zero/arr.length).toFixed(6))
}


plusMinus(1,1, 0, -1,-1)